cmd/ci-operator-prowgen/main.go

```diff
@@ -91,26 +96,104 @@ func generatePodSpec(org, repo, configFile, target string, additionalArgs ...str
 		ServiceAccountName: "ci-operator",
 		Containers: []kubeapi.Container{
 			{
-				Image:   "ci-operator:latest",
-				Command: []string{"ci-operator"},
-				Args:    append([]string{"--artifact-dir=$(ARTIFACTS)", fmt.Sprintf("--target=%s", target)}, additionalArgs...),
-				Env:     []kubeapi.EnvVar{{Name: "CONFIG_SPEC", ValueFrom: &configMapKeyRef}},
-				Resources: kubeapi.ResourceRequirements{
-					Requests: kubeapi.ResourceList{"cpu": *resource.NewMilliQuantity(10, resource.DecimalSI)},
-					Limits:   kubeapi.ResourceList{"cpu": *resource.NewMilliQuantity(500, resource.DecimalSI)},
-				},
+				Image:     "ci-operator:latest",
+				Command:   []string{"ci-operator"},
+				Args:      append([]string{"--artifact-dir=$(ARTIFACTS)", fmt.Sprintf("--target=%s", target)}, additionalArgs...),
+				Env:       []kubeapi.EnvVar{{Name: "CONFIG_SPEC", ValueFrom: &configMapKeyRef}},
+				Resources: containerResourceReqs,
 			},
 		},
 	}
 }
 
+func generatePodSpecCluster(org, repo, branch, target, release string, test *cioperatorapi.TestStepConfiguration) *kubeapi.PodSpec {
+	var template, clusterType string
+	if conf := test.OpenshiftAnsibleClusterTestConfiguration; conf != nil {
```

Comment:
Might want to be a `switch` and handle the `default` with an error

Reply:
Added in openshift/ci-operator#125.